src/pytest_inmanta_lsm/util.py

```diff
@@ -0,0 +1,83 @@
+"""
+    Pytest Inmanta LSM
+
+    :copyright: 2024 Inmanta
+    :contact: code@inmanta.com
+    :license: Inmanta EULA
+"""
+
+import asyncio
+import collections.abc
+import typing
+
+
+class MultiException(RuntimeError):
```

Comment:
Using https://docs.python.org/3/library/exceptions.html#ExceptionGroup would be cool, but that is only from 3.11 I don't know which versions we target here?